apps/example/src/components/auth/HeaderUser.tsx

```diff
@@ -9,7 +13,7 @@ import { Stack, Button } from "@cloudoperators/juno-ui-components"
 import Avatar from "./Avatar"
 import { useAuthLoggedIn, useAuthData } from "../StoreProvider"
 
-const HeaderUser = ({ login, logout }) => {
+const HeaderUser = ({ login, logout }: any) => {
```

Comment:
can't we make hier functions?
https://github.com/cloudoperators/juno/blob/main/packages/oauth/src/mockedSession.ts#L64-L72